blocks/test/parser.js

```diff
@@ -75,6 +75,7 @@ describe( 'block parser', () => {
 				'<!-- wp:core/unknown-block -->Ribs<!-- /wp:core/unknown-block -->';
 
 			expect( parse( postContent ) ).to.eql( [ {
+				uid: '1',
```

Comment:
This seems fragile as it relies on `uniqueId` never having been called before in the current process. Instead it might be better to test that it's numeric (or even just assigned), which will probably require breaking apart the assertions to:


```js
expect( parsed[ 0 ].uid ).to.be.a( 'number' );
// ... etc
```